tests/datapackages/2018-19/epre/epre.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
I meant we should have tests that run against the [actual schema](https://raw.githubusercontent.com/vulekamali/data-checks/master/schema/epre.json), so the test datapackages should refer to the actual schema

Reply:
@jbothma the problem is that the datapackage `.json` file must be in the same directory or a higher-directory level than the schema. So the path to the schema can't be, for example, `../../schema/epre.json`. I'm looking at whether it would be possible to create a symbolic link to the schema and then use the path to the symbolic link.

Reply:
I'll just use the url to schema :)

Reply:
yeah - it's a bit of a pain - that also makes it hard to test out a change in the schema - you end up having to modify the branch name in the URLs.

Let's keep this in mind.